bank_component.rb

```diff
@@ -0,0 +1,97 @@
+=begin
+ Feature 1: 
+ This feature will allow user to either sign up for an account.
+ Or log in with their last name,last 4 digits of their social security number, and password.        
+=end
+
+
+
+#class for log in and sign up method
+class LogInSignUp
+
+    @@user_info=[
+        {first:'andy', last:'park', password:'hello123', ssn:'123456789'},
+        {first:'Jason', last:'kim', password:'bye123', ssn:'135791098'}
+    ]
+
+    #Instance variable to determine if the user's credential is validated
+    @user_credential_validation = false
+    
+    attr_accessor :last_name, :password, :ssn
+
+    def initialize(last_name, password, ssn)
+        @last_name = last_name
+        @password=password
+        @ssn=ssn
```

Comment:
Nit: spaces around the assignment operator

("Nit" as in "nitpick". It's part of the culture to label our nitpicks as such, so the author doesn't feel bashed. As in, "we recognize this is not a critical thing to fix, but we still want to give you a friendly reminder")